index.ts

```diff
@@ -217,6 +217,17 @@ function shouldCustomerBeTracked(customer: Customer, eventsConfig: EventsConfig)
     }
 }
 
+function filterByByteLength(obj: Record<string, any>, maxLength: number): Record<string, any> {
+    const filtered: Record<string, any> = {}
+    for (const [key, value] of Object.entries(obj)) {
+        if (typeof value === 'string' && Buffer.byteLength(value) > maxLength) {
+            continue
```

Comment:
I feel like this leaves a lot of room for confusion because there will be no indication to the customer that the length is too long. It simply won't exist. I feel like it makes more sense to truncate it somehow.